### ISSUE TITLE
Display Rewards Breakdown to users

### ISSUE THREAD

**Issue by wolf31o2** [TosiDrop/vm-frontend#418]
The Vending Machine API supports a rewards breakdown, which returns detailed information on the rewards given, including when they were given, when they expire, and who gave the rewards to the user.

This information should be shown to users so tokens don't "disappear" on them when unclaimed and they expire. Here's the breakdown on VM.

<img width="1647" alt="Screen Shot 2023-03-09 at 8 58 50 AM" src="https://user-images.githubusercontent.com/380021/224047794-2bec171c-52fc-4b7b-90e3-f35493619e7a.png">

**Comment by reqlez** [comment]
Yes! This was actually a popular request from users. People want to know when they can "accumulate" and "what".

I think this is definitely something worth working on.
